neurodamus/gap_junction.py

```diff
@@ -2,14 +2,12 @@
 Main module for handling and instantiating synaptical connections
 """
 from __future__ import absolute_import
-import logging
 import numpy as np
 from os import path as ospath
 
 from .connection_manager import ConnectionManagerBase
 from .core.configuration import ConfigurationError
-from .io.synapse_reader import SynapseReader, SynReaderNRN, SonataReader, SynapseParameters, \
-    FormatNotSupported
+from .io.synapse_reader import SynapseReader, SonataReader, SynapseParameters
```

Comment:
Is `SynapseReader` needed still, then?

Reply:
Good point. In fact,  it is used for assigning  `CONNECTIONS_TYPE`
```
CONNECTIONS_TYPE = SynapseReader.GAP_JUNCTIONS
``` 
But it seems that this variable is not doing any thing. I will try to remove it.